pkgs/applications/video/invidtui/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, buildGoModule, fetchFromGitHub, yt-dlp, ffmpeg, mpv }:
+
+buildGoModule rec {
+  pname = "invidtui";
+  version = "0.3.6";
+
+  src = fetchFromGitHub {
+    owner = "darkhz";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-zUr0zrIJPpqhHvL7PFFN7cgcgBXV+WHO/eRes7+HzxM=";
+  };
+
+  vendorHash = "sha256-cKvY3/3N3SESBVol7Af3M3mJaPwxLzd/rKN8P+qh7sY=";
+
+  doCheck = false;
+
+  propagatedBuildInputs = [ yt-dlp ffmpeg mpv ];
```

Comment:
```suggestion
```
This shouldn't be needed since the program is patched with absolute paths to these